app/code/Magento/Catalog/Model/ResourceModel/Category.php

```diff
@@ -1078,7 +1078,6 @@ public function countVisible()
      */
     public function load($object, $entityId, $attributes = [])
     {
-        $this->_attributes = [];
```

Comment:
That was static test that property was declared dynamically and actually never used.